src/Core/Children.js

```diff
@@ -41,62 +41,50 @@ function childrenArray (children) {
  * @param {function} callback
  * @param {*} thisArg
  */
-function childrenEach (children, callback, thisArg) {
+function eachChildren (children, callback, thisArg) {
 	if (children != null)
```

Comment:
should this be an `assert`?

Reply:
I don't understand the question?

Reply:
As in, this function should never be called with `children = null`, and if so, it implies an error somewhere else in the code.
i.e. throw if `children != List()`

Reply:
This is for the following.

```
Children.forEach(this.props.children)
```

Since `this.props.children` is an opaque data-structure i could be a element, undefined or an array, this deals with the undefined/null case like the way React deals with it.